tools/alive-tv.cpp

```diff
@@ -238,7 +245,9 @@ bool compareFunctions(llvm::Function &F1, llvm::Function &F2,
 }
 
 void optimizeModule(llvm::Module *M) {
-  llvm::LoopAnalysisManager LAM;
+  LLVMOptimizer optimizer(optPass);
+  optimizer.optimizeModule(M);
+  /*llvm::LoopAnalysisManager LAM;
```

Comment:
probably don't comment out blocks of code like this -- either leave it as live code if we might want it or else remove it

Reply:
fixed!